xcdat/spatial.py

```diff
@@ -158,47 +163,50 @@ def average(
         dataset[dv.name] = self._averager(dv, axis, dv_weights)
         return dataset
 
-    def _validate_axis(
+    def _validate_axis_arg(
         self, data_var: xr.DataArray, axis: Union[List[SpatialAxis], SpatialAxis]
     ) -> List[SpatialAxis]:
-        """Validates if ``axis`` arg is supported and exists in the data var.
+        """Validates the ``axis`` dimension exists in the data variable.
 
         Parameters
         ----------
         data_var : xr.DataArray
             The data variable.
         axis : Union[List[SpatialAxis], SpatialAxis]
-            List of axis dimensions or single axis dimension to average over.
+            List of axis dimensions or single axis dimensions.
```

Comment:
I like how you wrote this below: "List of axis dimensions or a single axis dimension"

Reply:
I meant to use the bottom description, thanks for the catch